Podfile

```diff
@@ -12,6 +12,10 @@ target 'ios-base' do
   pod 'Crashlytics', '~> 3.12.0'
   pod 'R.swift', '~> 5.0.3'
 
+  # Uncomment if needed ---
+  # pod 'PagedLists', '~> 1.0.0'
+  # ------
```

Comment:
Should we have this commented here? Or maybe it is enough with the mention added under Utilities?

Reply:
I thought it would be cool to promote the libraries here as well. The README it something you don't look after installing it after all.